### ISSUE TITLE
Tree view reponsive issue

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#7919]
### Bug description

The tree view doesn't work great on mobile devices yet. Probably due to the `Save changes` button. It causes overflow and the icons in the segmented control to disappear. 

Perhaps the control + save button should go on a new line?
![IMG_1378](https://user-images.githubusercontent.com/69107412/232288308-ac2e4529-d674-496d-a93e-334243e37772.PNG)

### How to reproduce

.

### Environment

```yaml
alpha-3
```

### Installation

Starter Kit using via CLI

### Antlers Parser

None

**Comment by jasonvarga** [comment]
Not v4 specific.

**Comment by robdekort** [comment]
That is true. Guess I labeled it since v4 comes with proper mobile views :).

**Comment by jackmcdade** [comment]
Took a stab at this and there is no perfect CSS-only solution with the current UI. It's gonna need a bit of a redesign.

**Comment by robdekort** [comment]
Yeah, I was afraid of that. Let me know if you need to bounce some ideas off of me.

**Comment by duncanmcclean** [comment]
This seems to be fixed on v6! 🎉
